api/types/container/config.go

```diff
@@ -32,27 +33,7 @@ type StopOptions struct {
 	Timeout *int `json:",omitempty"`
 }
 
-// HealthConfig holds configuration settings for the HEALTHCHECK feature.
-type HealthConfig struct {
-	// Test is the test to perform to check that the container is healthy.
-	// An empty slice means to inherit the default.
-	// The options are:
-	// {} : inherit healthcheck
-	// {"NONE"} : disable healthcheck
-	// {"CMD", args...} : exec arguments directly
-	// {"CMD-SHELL", command} : run command with system's default shell
-	Test []string `json:",omitempty"`
-
-	// Zero means to inherit. Durations are expressed as integer nanoseconds.
-	Interval      time.Duration `json:",omitempty"` // Interval is the time to wait between checks.
-	Timeout       time.Duration `json:",omitempty"` // Timeout is the time to wait before considering the check to have hung.
-	StartPeriod   time.Duration `json:",omitempty"` // The start period for the container to initialize before the retries starts to count down.
-	StartInterval time.Duration `json:",omitempty"` // The interval to attempt healthchecks at during the start period
-
-	// Retries is the number of consecutive failures needed to consider a container as unhealthy.
-	// Zero means inherit.
-	Retries int `json:",omitempty"`
-}
+type HealthConfig = dockerspec.HealthcheckConfig
```

Comment:
As this package is consumed by others, should we keep some basic GoDoc?

```suggestion
// HealthConfig holds configuration settings for the HEALTHCHECK feature.
// It is an alias for [dockerspec.HealthcheckConfig].
type HealthConfig = dockerspec.HealthcheckConfig
```

Reply:
(not 100% sure if we need to add the "is an alias for", as some of that could potentially change in future if the OCI would accept it to be added to the spec).

Reply:
Ah, good one. I think we can omit the "is an alias for" because that's already what that code line explicitly does.

Reply:
Added the simple godoc.

Reply:
Thanks! Sorry for the nit-pick, but wanted to avoid the code-churn ("remove the line, and adding it back again in a follow-up") on this one 👍

Reply:
Definitely a valid comment! I didn't intend to actually remove it, so thanks for bringing this one up 😄